src/other/utils.jl

```diff
@@ -221,16 +221,22 @@ end
 Equivalent to `Threads.@spawn` if `threads === true`,
 otherwise run `expr` and return a `Task` that returns its value.
 """
-macro spawn_or_run_task(threads, expr)
-    letargs = Base._lift_one_interp!(expr)
+macro spawn_or_run_task(threads, ex)
+    letargs = Base._lift_one_interp!(ex)
 
-    thunk = esc(:(()->($expr)))
+    thunk = :(()->($(esc(ex))))
+    @static if VERSION >= v"1.10.0-DEV"
```

Comment:
why `@static` here?

Reply:
I tend to always use it for version checks (other than in global scope) since [that ensures the branch is eliminated at parse time](https://stackoverflow.com/a/65740489/2413179). But in this case since it's a macro I'm not sure it makes a difference. Maybe I'm being overzealous. I can drop that if you prefer -- or conversely use it with the other check for consistency.

Reply:
This is the reason I asked - it is there in one place but not in the other. I would assume that `@static` is not needed, but I asked on Slack.